dev-docs/bidders/zmaticoo.md

```diff
@@ -39,3 +39,97 @@ To use this bidder you will need a valid pubId. For further information, please
 | `bapp`               | optional | Block list of applications by their platform-specific exchange independent application identifiers                         | `badv: ["com.blocked"]` | `string array` |
 | `source`             | optional | A Source object that provides data about the inventory source and which entity makes the final decision (See OpenRTB spec) | `source: {}`            | `object`       |
 | `ext`                | optional | An extension object that allows for custom fields and objects to be sent  (See OpenRTB spec)                               | `ext: {}`               | `object`       |
+
+### AdUnit Format Example
+#### AdUnit Format for Banner
+
+```javascript
+ var adUnits = [{
+    code: 'banner-01',
+    mediaTypes: {
+        banner: {
+            sizes: [[320, 50]]
+        }
+    },
+    bids: [{
+        bidder: "zmaticoo",
+        bidId: '100001222',
+        params: {
+            pubId: 'prebid-test',
+            tagid:"001",
+            bidfloor:1,
+            device: {
+
+            }
+        },
+    }]
+}];
+```
+
+#### AdUnit Format for video
+
+```javascript
+     var adUnits = [{
+    code: 'video_ad_unit_1',
+    mediaTypes: {
+        video: {
+            playerSize: [480, 320],
+            mimes: ['video/mp4'],
+            context: 'outstream',
+        }
+    },
+    renderer: {
+        url: 'https://creative.cdnyeah.com/tpl/renderer/video/zmaticooOutsteam.js',
+        render: function (bid) {
+            adResponse = {
+                ad: {
+                    video: {
+                        content: bid.vastXml,
+                        player_height: 320,
+                        player_width: 480,
+                    },
+                },
+            }
+            bid.renderer.push(() => {
+                ZMOutstreamVideo.renderAd({
+                    vastUrl: bid.vastXml,
+                    adUnitCode: bid.adUnitCode
+                });
+            });
+        }
+    },
+    bids: [{
+        bidder: "zmaticoo",
+        bidId: '100001222',
```

Comment:
Please remove bidId as a param or explain what it means -- normally bidIds are generated randomly for each auction.